addOns/jwt/src/main/resources/org/zaproxy/zap/extension/jwt/resources/Messages.properties

```diff
@@ -0,0 +1,78 @@
+jwt.settings.title=JWT Settings
+jwt.toolmenu.settings=JWT Settings...
+ascanrules.jwt.name=JWT Scanner 
+ascanrules.jwt.description=Scanner for finding vulnerabilities in JWT implementations.
+jwt.settings.rsa.header=<html><h3>RSA:<h3></html>
+jwt.settings.general.header=<html><h3>General:<h3></html>
+jwt.settings.rsa.trustStorePath=TrustStore path:
+jwt.settings.rsa.trustStorePassword=TrustStore password:
+jwt.settings.filechooser.button=Select...
+jwt.settings.general.ignoreClientSideScan.checkBox=Ignore Client Configuration Scan
+
+# Client side JWT storage vulnerabilities
+jwt.scanner.client.vulnerability.cookiehttponly.name=No HttpOnly Flag on cookie containing JWT token.
```

Comment:
```suggestion
jwt.scanner.client.vulnerability.cookiehttponly.name=No HttpOnly Flag on Cookie Containing JWT Token
```

Is "JWT Token" redundant? Is that kinda like saying "SSN Number"......'Social Security Number Number" ?

Reply:
If it is redundant as described it should be changed throughout.